src/main/java/step2/view/OutputView.java

```diff
@@ -11,62 +14,64 @@ public class OutputView {
 	private static final int MAX_WIN_COUNT = 6;
 	private static final int BENEFIT_STANDARD = 1;
 
-	public void resultView(HashMap<Integer, Integer> finalMatch) {
+	public void resultView(HashMap<Integer, Integer> totalCount, int bonusMatchCount) {
 		System.out.println("당첨 통계");
 		System.out.println("--------");
 
 		for (int key = MIN_WIN_COUNT; key <= MAX_WIN_COUNT; key++) {
-			showResult(key, isValid(finalMatch.get(key)));
+			showResult(key, bonusMatchCount, totalCount);
 		}
 	}
 
-	private int isValid(Integer matchSum) {
-		if (matchSum == null) {
-			return DEFAULT_COUNT;
-		}
-		return matchSum;
-	}
+	private void showResult(int key, int bonusMatchCount, HashMap<Integer, Integer> totalCount) {
+		int countOfMatch = Rank.of(key).getCountOfMatch();
 
-	private void showResult(int key, int matchSum) {
-		if (key == MIN_WIN_COUNT) {
-			System.out.println("3개 일치" + "(5000원)- " + matchSum + "개");
-		}
-		if (key == MIN_WIN_COUNT + 1) {
-			System.out.println("4개 일치" + "(50000원)- " + matchSum + "개");
-		}
-		if (key == MIN_WIN_COUNT + 2) {
-			System.out.println("5개 일치" + "(1500000원)- " + matchSum + "개");
-		}
-		if (key == MAX_WIN_COUNT) {
-			System.out.println("6개 일치" + "(2000000000원)- " + matchSum + "개");
+		if (countOfMatch == SECOND.getCountOfMatch()) {
+			rankSecondMessage(bonusMatchCount, countOfMatch);
+			return;
 		}
+
+		System.out.printf("%s개 일치" + "(%d)- " + totalCount.get(key) + "개\n", countOfMatch,
+			Rank.of(key).getWinningMoney());
 	}
 
 	public void showTotalTicket(int tickets) {
 		System.out.println(tickets + "매를 구매했습니다");
 	}
 
 	public void showLotteries(int tickets, Lotteries lotteries) {
-		for (int i = 0; i < tickets; i++) {
-			System.out.println(Arrays.toString(lotteries.showLotteries(i).getLotto().toArray()));
+		for (int i = DEFAULT_COUNT; i < tickets; i++) {
+			System.out.println(lotteries.showLotteries(i).getLotto());
 		}
 	}
 
 	public void showRate(double rate) {
-		if (checkRate(rate)) {
-			System.out.println("수익률은 " + String.format("%.2f", rate) + "입니다");
+		defaultMessage(rate);
+		if (isProfit(rate)) {
 			return;
 		}
-		System.out.println(
-			"수익률은 " + String.format("%.2f", rate) + "입니다(기준이 1이기 때문에 결과적으로 손해라는 의미임");
+		System.out.println("기준이 1이기 때문에 결과적으로 손해라는 의미임");
 	}
 
-	private boolean checkRate(double rate) {
+	private boolean isProfit(double rate) {
 		if (rate < BENEFIT_STANDARD) {
 			return false;
 		}
 		return true;
```

Comment:
다음처럼 줄여서 작성해볼 수 있을 것 같아요.
```suggestion
		return rate > BENEFIT_STANDARD;
```